charts/spire/charts/spire-server/templates/NOTES.txt

```diff
@@ -8,17 +8,14 @@ Installed {{ .Chart.Name }}…
 {{- if eq (.Values.tornjak.enabled | toString) "true" }}
 
 ### WARNING ###
+
 This Tornjak is configured without authentication and it is intended for
 testing only. Please do not use this version in production.
 
 Tornjak APIs (Backend):
       kubectl -n {{ include "spire-server.namespace" . }} port-forward {{ include "spire-server.fullname" . }}-0 10000:10000
 
-Tornjak UI (Frontend):
-      kubectl -n {{ include "spire-server.namespace" . }} port-forward {{ include "spire-server.fullname" . }}-0 3000:3000
-
-Tornjak API access:  {{ include "tornjak.apiURL" . }}
-Tornjak UI access:   {{ include "tornjak.FrontendURL" . }}
+Tornjak API access:  {{ include "spire-tornjak.apiURL" . }}
```

Comment:
Oh. I see. Its only used here....

Maybe this should include the full instructions to the
```
kubectl port-forward xxxx
open browser to http://localhost:xxxx
```
And the variable wouldn't be needed.